Cargo.toml

```diff
@@ -154,6 +154,7 @@ js-sys = "0.3.45"
 serde_json = "1.0"
 wasm-bindgen = "0.2.68"
 wasm-bindgen-futures = "0.4.18"
+wasm-streams = { version = "0.2", optional = true }
```

Comment:
May be try bumping `wasm-streams` to `0.3`. https://github.com/MattiasBuelens/wasm-streams/blob/main/CHANGELOG.md

Reply:
I don't think this should be part of this MR. It doesn't fix anythong related to this feature right?